big_tests/tests/muc_light_SUITE.erl

```diff
@@ -321,15 +321,6 @@ disco_info(Config) ->
 disco_info_with_mam(Config) ->
     disco_features_story(Config, true).
 
-disco_info_story(Config, HasMAM) ->
```

Comment:
This function should be used in the two test cases above instead of `disco_features_story`.

Reply:
Done. Thanks.